taskcluster/rb_taskgraph/signing.py

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Apply some defaults and minor modifications to the jobs defined in the build
+kind.
+"""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from taskgraph.transforms.base import TransformSequence
+from taskgraph.util.treeherder import inherit_treeherder_from_dep
+
+
+transforms = TransformSequence()
+
+
+@transforms.add
+def stuff(config, tasks):
+    for task in tasks:
+        dep = task.pop("primary-dependency")
+        task["dependencies"] = {"build": dep.label}
+        task["name"] = dep.label[len(dep.kind) + 1 :]
```

Comment:
The name is the subset of the label, but the first part of the label is the kind? :thinking: Should there be a property separate from `label` so you don't need to do any manual parsing here?